Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner =()=>
+     (
+      <div>
+        <img className="bannerImage" src ={"https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png"} alt=""></img>
```

Comment:
```suggestion
        <img className="bannerImage" src ="https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png" alt=""></img>
```
You only need curly brackets if you're doing some dynamic javascript inside the brackets. (i.e. referening state or a variable)

Reply:
Also, using a div with a background image on this particular image would have allowed you to get much closer to pixel perfect on this image. :)